Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/data/source/InMemoryLocalTaskDataSource.java

```diff
@@ -22,12 +22,12 @@
     private Tasks localTasks = Tasks.asSynced(
             ImmutableList.<Task>of(
                     Task.builder()
-                        .id(Id.from("42"))
```

Comment:
Was this value changed for any particular reason? Or just don't like the meaning of "life, the universe and everything?" 👅